cmd/auth-rest/startcmd/start.go

```diff
@@ -44,6 +51,28 @@ const (
 	logLevelPrefixFlagUsage = "Default logging level to set. Supported options: CRITICAL, ERROR, WARNING, INFO, DEBUG." +
 		`Defaults to info if not set. Setting to debug may adversely impact performance. Alternatively, this can be ` +
 		"set with the following environment variable: " + logLevelEnvKey
+
+	databaseTypeFlagName      = "database-type"
+	databaseTypeEnvKey        = "DATABASE_TYPE"
+	databaseTypeFlagShorthand = "d"
+	databaseTypeFlagUsage     = "The type of database to use for storage. Supported options: mem, couchdb. " +
```

Comment:
nitpick: this could have been included in `database-url` with a separator

Reply:
True... however, all the other projects have separate flags (since that's how I set it up originally), so for consistency I think we have to maintain separate flags for now.

Reply:
@DRK3 improvements are always welcome... can't let "consistency" hold us back

Reply:
Also true. I'm just not sure that the benefits outweigh the costs here. Is it really worth breaking consistency just for the sake of one less command line flag... which also necessitates introducing some sort of "separator" syntax. Seems like a lateral move to me.

Reply:
The norm "out there" is to have the database URL encode the DB driver. If you look at Hydra's DSN parameter, you'd understand. Same pattern holds in other programming languages. Having a flag to indicate the DB type and another the URL seems out of the norm.

Reply:
In the future when/if I'm doing refactoring work, I'll take a look into seeing how I can combine these database parameters into one flag... I'll reach out to you then to see what the best format would be. I certainly want to avoid inventing a new pattern :)